telco-examples/edge-clusters/README.md

```diff
@@ -138,11 +137,10 @@ The first thing is to modify the `telco-capi-single-node.yaml` file and replace
 - `${NON-ISOLATED_CPU_CORES}` - The cores listed could be used shared for the rest of the process running on the edge cluster. For example, `0,20,21,39` could be used for the non-isolated cores.
 - `${CPU_FREQUENCY}` - The frequency to be used for the CPU cores. For example, `2500000` represents 2.5Ghz configuration and it could be used to set the CPU cores to the max performance.
 - `${DPDK_PCI_ADDRESS}` - The PCI device to be used for the DPDK configuration. You could get that info using `lspci` command to list the PCI devices and then selecting the device to be used for the DPDK configuration. For example, `0000:00:1f.6` could be used for the DPDK configuration.
-- `${RKE2_VERSION}` - The RKE2 version to be used for the edge cluster. For example, `v1.28.3+rke2r1` could be used for the edge cluster.
 
 You can also modify any other parameter in the `telco-capi-single-node.yaml` file to match with your requirements e.g. DPDK configuration, number of VFs to generate, number of SRIOV interfaces, etc. This is basically a template to be used for the edge cluster deployment. 
 
-** Note: Remember to locate the `eibimage-slemicro55rt-telco.raw` file generated in [Create the image for the edge cluster](#create-the-image-for-the-edge-cluster) into the management cluster httpd cache folder to be used during the edge cluster provision step.
+** Note: Remember to locate the `eibimage-slmicro60rt-telco.raw` file generated in [Create the image for the edge cluster](#create-the-image-for-the-edge-cluster) into the management cluster httpd cache folder to be used during the edge cluster provision step.
```

Comment:
We don't have to change it in this PR, but in future we could rename this e.g just to `eibimage.raw` or similar to reduce the things we need to update in future

Reply:
Although actually if we want to enable multiple versions in the imagecache perhaps we need to keep this convention 🤔